includes/class-wcpdf-install.php

```diff
@@ -61,8 +61,8 @@ public function do_install() {
 	 * Plugin install method. Perform any installation tasks here
 	 */
 	protected function install() {
-		// only install when php 5.3 or higher
-		if ( version_compare( PHP_VERSION, '5.3', '<' ) ) {
+		// only install when php 7.1 or higher
```

Comment:
Plugin should install from 5.6+, otherwise the people that use the 0.8.3 extension miss out on the installation  procedure